src/libclipper/include/clipper/containers.hpp

```diff
@@ -31,13 +32,14 @@ class ModelContainer {
 
   size_t get_batch_size(Deadline deadline);
   double get_average_throughput_per_millisecond();
-  void update_throughput(size_t batch_size, long total_latency);
+  void update_container_stats(size_t batch_size, long total_latency);
   void send_feedback(PredictTask task);
 
   VersionedModelId model_;
   int container_id_;
   int replica_id_;
   InputType input_type_;
+  clipper::metrics::Histogram latency_hist_;
```

Comment:
This seems useful! Let's keep it. See previous comment about separating it from the `update_throughput` method.